addons/account/i18n/account.pot

```diff
@@ -12320,8 +12320,7 @@ msgstr ""
 #: code:addons/account/models/account_move.py:0
 #, python-format
 msgid ""
-"You cannot use this general account in this journal, check the tab 'Entry "
-"Controls' on the related journal."
+"You cannot use %s in %s, check the tab 'Entry Controls' on the related journal."
```

Comment:
Please don't do that in stable when it's only for a small improvement like here. 
I can do it in 13.5 when forward-porting your commit, so that it's there in v14 if you want :)

Reply:
Ok for me (feel free to do the forward port dance), although I think it is also a bug which needs to be fixed in stable as the user has absolutely no clue what is wrong with the UserError given by now (almost similar to a stack trace error). 

It is worth to kill translations in this case in stable IMO.

@mart-e 
Any opinion besides that we break this term in translations and you have some extra work that it won't break the error message due to the missing %s in a translation? 😉

Reply:
It's always a tradeoff. The majority of our databases on the SaaS are in another language than English, is the fix worth having a message they potentially won't understand? As said Olivier, we avoid breaking error messages if we can. Adding a new message for uncovered usecases is a valid option though.

Reply:
Alright, I will rework and provide two PRs to get in this into stable and the next release. 

Not sure, I got the "worth having a message they potentially won't understand" as the current messages is a general message for a specific issue which should be named by the account and the journal name, so it is worth IMO to have the translators (including myself) to redo the translations properly adapted to the new term.